### ISSUE TITLE
Add additional RBAC roles

### ISSUE THREAD

**Issue by kirbsauce** [kubecost/cost-analyzer-helm-chart#1164]
**What problem are you trying to solve?**
I'm trying to allow users to create/save reports but not be able to perform and admin tasks (in the settings).

**Describe the solution you'd like**
A new role to satisfy "read-only + saved reports" capabilities.

**Describe alternatives you've considered**
Currently, we need to give everyone admin access and instruct them to not visit Settings

**How would users interact with this feature?**
The same way in which we interact with current roles.

**Comment by AjayTripathy** [comment]
Hey all, here's a quick update on how we're thinking about our permissions model as well as what's implemented so far and what's TODO. Feedback welcome!

Support three predefined Groups:
  Admin: 
    See all clusters
    See all namespaces
    See all assets 
    Edit settings 
    Edit saved reports and alerts
    Kick off manual ETL rebuilds
  Reader:
    See namespaces scoped by custom groups
    See namespaces scoped by custom groups
    See assets scoped by custom groups [TODO]
    Deny manual ETL rebuilds with error message [TODO]
  Editor [TODO]
    See namespaces scoped by custom groups
    See assets scoped by custom groups
    Edit saved reports and alerts, scoped by custom groups
    Deny manual ETL rebuilds with error message [TODO]
    Support Filtering on Custom Groups
    Cluster-prefixes-to-users mapping
    Namespace-prefixes-to-users mapping
    Accounts-to-users mapping [TODO]

**Comment by kirbsauce** [comment]
@AjayTripathy , is there a duplicate under "Reader".

**Comment by AjayTripathy** [comment]
Yes, sorry! I'll link a full google doc with the design that I've been editing.  https://docs.google.com/document/d/1wdtxebOWv5626LL6I-19BQvP4BmQ4L-GTckKfn-2_ak/edit?usp=sharing

**Comment by kirbsauce** [comment]
@AjayTripathy , added the functionality of "denying users based on role" to the document.

**Comment by AdamStack18** [comment]
Pull Request: https://github.com/kubecost/kubecost-cost-model/pull/782  

@kaelanspatel Closing this issue out for us. Good work.
